conf/views/includes/footer.njk

```diff
@@ -9,7 +9,7 @@
         text: messages("footer.links.cookies")
       },
       {
-        href: "/accessibility-statement/manage-transit-movements",
+        href: "/accessibility-statement/check-transit-guarantee-balance",
```

Comment:
Would this remain as transit if the service name is updated in the statement to align with what is rendered in staging?

Reply:
I believe it's the file name of the statement itself